src/s1reader/s1_orbit.py

```diff
@@ -255,6 +257,58 @@ def get_orbit_file_from_dir(zip_path: str, orbit_dir: str, auto_download: bool =
 
     # search for orbit file
     orbit_file_list = glob.glob(os.path.join(orbit_dir, 'S1*.EOF'))
+
+    orbit_file = get_orbit_file_from_list(zip_path, orbit_file_list)
+
+    if not orbit_file:
+        orbit_dir = os.path.dirname(orbit_dir)
+        if not orbit_dir:
```

Comment:
`os.path.isdir(orbit_dir)` was tested above. What is the case that you are testing here?

Reply:
Good catch. Agree that the test here is not necessary. Removing the lines here.